org.filezillaproject.Filezilla.metainfo.xml

```diff
@@ -59,6 +61,7 @@
   <screenshots>
     <screenshot type="default">
       <image>https://raw.githubusercontent.com/flathub/org.filezillaproject.Filezilla/master/screenshot.png</image>
```

Comment:
since you are changing things, this should NEVER be the tip of a branch.

Reply:
Agreed